tiled/_tests/test_websockets.py

```diff
@@ -33,6 +35,7 @@ def tiled_websocket_context(tmpdir):
         yield context
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Requires Redis service")
```

Comment:
This can be done at the module level, I believe. Less work for Future Us when we add more tests or if we add Windows support.

Reply:
Sounds good, updated